packages/access-api/migrations/0004_add_accounts_table.sql

```diff
@@ -0,0 +1,21 @@
+-- Migration number: 0004 	 2023-01-24T15:09:12.316Z
+CREATE TABLE
+    IF NOT EXISTS accounts (
+        did TEXT NOT NULL PRIMARY KEY,
+        inserted_at TEXT NOT NULL DEFAULT (strftime ('%Y-%m-%dT%H:%M:%fZ', 'now')),
+        updated_at TEXT NOT NULL DEFAULT (strftime ('%Y-%m-%dT%H:%M:%fZ', 'now')),
+        UNIQUE (did)
+    );
+
+CREATE TABLE
+    IF NOT EXISTS delegations (
+        cid TEXT NOT NULL PRIMARY KEY,
+        bytes BLOB NOT NULL,
+        audience TEXT NOT NULL,
+        issuer TEXT NOT NULL,
+        expiration TEXT,
+        inserted_at TEXT NOT NULL DEFAULT (strftime ('%Y-%m-%dT%H:%M:%fZ', 'now')),
+        updated_at TEXT NOT NULL DEFAULT (strftime ('%Y-%m-%dT%H:%M:%fZ', 'now')),
+        UNIQUE (cid),
+        FOREIGN KEY (audience) REFERENCES accounts (did) ON UPDATE CASCADE ON DELETE CASCADE
```

Comment:
If we change something in accounts, delegation stays in sync. Its just a good default so we dont store dangling delegations because these are not issued by us. 
Updates any change to PK/FK pair (should never happen) and deletes delegations if we delete account.